### DISCUSSION TITLE
Enable gtiflow process

### DISCUSSION THREAD

**Discussion by neonima** [devcherchecollegue-org/rust-pong#9] [Ideas]
As proposed by @Westixy in #7 

From Westixy
For the moment we only have a master branch, for this kind of project it could be enough.
As it is a learning project, it could be fun to make a full gitflow strategy like so : https://www.fredonism.com/a-practical-take-on-gitflow-and-semantic-versioning .

I have a counter-argument to this, it might be harder for new contributors, who don't know how gitflow works, to get in the project. So maybe thats not a good idea.

What do you think ?

**Comment by Fred-si** [comment]
Salut.

Perso je suis ni pour ni contre (aucune idée de comment traduire ça en anglais)
